public/index.html

```diff
@@ -25,7 +25,7 @@
             <img alt="RChain Community" src="https://avatars1.githubusercontent.com/u/43198800?s=88&v=4">
           </a>
         </div>
-        <div class="progress" style="width: 50%">
+        <div class="progress" style="width: 20%">
```

Comment:
what does width have to do with ballotexample.json and abstain accts?

Reply:
this was an attempt to fix screw up on narrow screen covering heading.

Reply:
I don't see how your answer is responsive to my question, so I re-iterate: what does width have to do with this pull request, which is about ballotexample.json and abstain accts?